src/2-spatial-data-preparation.py

```
@@ -281,84 +325,98 @@ def mask(block, input_nodata, template_nodata):
-                        with rioxarray.open_rasterio(vrt, chunks = {'x': chunkDims, 'y': chunkDims}, lock = False) as covariateRaster:
-                            
+                        with rioxarray.open_rasterio(vrt, chunks={'x': chunkDims, 'y': chunkDims}, lock=False) as covariateRaster:
+
                             # Check that covariate layer overlaps template
                             # - Note that bounds are ordered xmin, ymin, xmax, ymax
                             tb = templateBounds
                             cb = covariateRaster.rio.bounds()
                             if cb[0] > tb[0] or cb[1] > tb[1] or cb[2] < tb[2] or cb[3] < tb[3]:
-                                raise ValueError(print("The extent of the", covariateDataSheet.CovariatesID[i], 
+                                raise ValueError(print("The extent of the", covariateDataSheet.CovariatesID[i],
                                                        "raster does not overlap the full extent of the template raster. Ensure all covariate rasters overlap the template extent before continuing."))
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)